providers/bnxt_re/verbs.c

```diff
@@ -1012,11 +1012,13 @@ static int bnxt_re_check_qp_limits(struct bnxt_re_context *cntx,
 static void bnxt_re_free_queue_ptr(struct bnxt_re_qp *qp)
 {
 	if (qp->jrqq) {
-		free(qp->jrqq->hwque);
+		if (qp->jrqq->hwque)
```

Comment:
Why did you add these checks in all places?
https://linux.die.net/man/3/free
If ptr is NULL, no operation is performed.

Reply:
fixed